packages/amplify-category-api/src/index.ts

```diff
@@ -315,26 +317,32 @@ export const transformCategoryStack = async (context: $TSContext, resource: $TSO
       const backendDir = pathManager.getBackendDirPath();
       const overrideDir = path.join(backendDir, resource.category, resource.resourceName);
       const isBuild = await buildOverrideDir(backendDir, overrideDir).catch(error => {
-        printer.error(`Build error : ${error.message}`);
-        throw new Error(error);
+        throw new AmplifyError('InvalidOverrideError', {
+          message: error.message,
+          link: 'https://docs.amplify.aws/cli/graphql/override/',
+        });
       });
-      await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'compileSchema', [
-        context,
-        {
-          forceCompile: true,
-          overrideConfig: {
-            overrideFlag: isBuild,
-            overrideDir,
-            resourceName: resource.resourceName,
+      try {
+        await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'compileSchema', [
+          context,
+          {
+            forceCompile: true,
+            overrideConfig: {
+              overrideFlag: isBuild,
+              overrideDir,
+              resourceName: resource.resourceName,
+            },
           },
-        },
-      ]);
-    }
-  } else if (resource.service === AmplifySupportedService.APIGW) {
-    if (canResourceBeTransformed(resource.resourceName)) {
-      // Rebuild CFN
-      const apigwStack = new ApigwStackTransform(context, resource.resourceName);
-      apigwStack.transform();
+        ]);
+      } catch (error) {
+        throw new AmplifyErrorConverter().create(error);
```

Comment:
Is AmplifyErrorConverter stateless ?
If so I'd rather 
```
import * as AmplifyErrorConverter from './errors/amplify-error-converter';

AmplifyErrorConverter.convertToAmplifyError(error)
```

Reply:
If you intend to keep AmplifyErrorConverter as is you should rename `create` to `convert` .